ftw/upgrade/command/__init__.py

```diff
@@ -142,7 +144,25 @@ def __call__(self):
         args = self.parser.parse_args()
         configure_logging(args)
         setattr(args, 'parser', self.parser)
-        args.func(args)
+        if getattr(args, 'all_sites', False):
+            info = {}
+            while True:
+                try:
+                    with capture() as out:
+                        args.func(args)
+                    output = out.getvalue().strip()
+                    if getattr(args, 'json', False):
+                        # "[]" must be turned into [].
+                        output = json.loads(output)
+                    info[args.picked_site] = output
+                except StopIteration:
+                    break
+            if getattr(args, 'json', False):
+                print(json.dumps(info))
+            else:
+                print(info)
```

Comment:
Now with json we print everything here, and without json we print the output here, but print 'Acting on site X' in `jsonapi.py`. Let me move those lines over here as well so it is in one spot.
Done and rebased.